pkg/synopsysctl/constants.go

```diff
@@ -53,6 +53,7 @@ var nativeClusterType = clusterTypeKubernetes
 
 var baseURL = "https://raw.githubusercontent.com/blackducksoftware/releases/master"
 
+var blackduckChartRepository = "https://artifactory.internal.synopsys.com/artifactory/bds-hub-helm-snapshot-local/blackduck/blackduck-2020.4.0.tgz"
```

Comment:
You could split this into a baseBlackDuckRepository variable so that we don't have to edit each function if it changes
This could be helpful if we break --chart-location-path into separate flags for --chart-name --chart-repo --chart-version